gateway/services/print.go

```diff
@@ -0,0 +1,26 @@
+package services
+
+import (
+	"github.com/HackIllinois/api/gateway/config"
+	"github.com/HackIllinois/api/gateway/middleware"
+	"github.com/arbor-dev/arbor"
+	"github.com/justinas/alice"
+	"net/http"
+)
+
+var PrintURI = config.PRINT_SERVICE
```

Comment:
This formatting is correct. We use the more standard camelCase formatting in the gateway instead of snake_case. `PrintURL` though instead of `PrintURI`

Reply:
Okay, so in that case it should be `PrintUri`.